instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -326,14 +326,30 @@ def collect_custom_response_headers_attributes(response_headers):
     return attributes
 
 
-def _parse_status_code(resp_status):
+def parse_status_code(resp_status):
```

Comment:
I would still keep this and other functions private. They are not intended to be user accessible. If there is really need we can always change from private to public but not vice versa.